lib/node_modules/@stdlib/stats/base/dists/levy/mean/benchmark/benchmark.js

```diff
@@ -42,8 +42,8 @@ bench( pkg, function benchmark( b ) {
 	mu = new Float64Array( len );
 	c = new Float64Array( len );
 	for ( i = 0; i < len; i++ ) {
-		mu[ i ] = ( randu() * 100.0 ) - 50.0;
-		c[ i ] = ( randu() * 20.0 ) + EPS;
+		mu[ i ] = uniform( EPS, 10.0 );
```

Comment:
```suggestion
		mu[ i ] = uniform( -50.0, 50.0 );
```

Please retain the original value ranges.